### ISSUE TITLE
Project Setup

### ISSUE THREAD

**Issue by Cadris** [SebLague/Neural-Network-Experiments#2]
Could You please add directions on how to set up this project and run it ?

**Comment by DiegoQuiros** [comment]
I completely agree. I'm familiar with Visual Studio, but I don't know how to run this project.  I would look for a solution file, but there is none.  Please help! Would love to be able to run it.

**Comment by DiegoQuiros** [comment]
This tutorial helps, but I still don't know how to run this project: https://www.youtube.com/watch?v=MawL2dQIZ7s

**Comment by DiegoQuiros** [comment]
1) Get Unity 2021.3.9f1 Personal
2) Download this project to your hard drive
3) Open Unity and then from the file menu choose "Open Project"
4) Point it to the directory where you downloaded this project
5) at the Top of Unity there should be a Play button. Hit it and you get a black window that does digit recognition
![Screenshot 2022-09-08 204349](https://user-images.githubusercontent.com/32005953/189267399-019f1172-6351-461d-a7dc-efec640d8cc3.jpg)

**Comment by Cadris** [comment]
Thanks for the reply. I will do that.
